phylogenetic/rules/construct_phylogeny.smk

```diff
@@ -18,7 +18,7 @@ rule tree:
         augur tree \
             --alignment {input.alignment:q} \
             --output {output.tree:q}
-          2> {log:q}
+          2>&1 | tee {log:q}
```

Comment:
[general commentary on E1 topology]

My E1 build is very different (running from `finish-rubella-7` with a local ingest) The difference is due to 5 strains forming an outgroup (consistently so) in my local runs: MT249313, OM735624, ON861827, MK780807, KU958641. In your [staging buid](https://next.nextstrain.org/staging/rubella/E1-new-ref) these strains fall within clade 1B. I haven't tracked down _why_. Did you see trees like this during development? 

<img width="603" alt="image" src="https://github.com/user-attachments/assets/8da9352d-6bc2-46ee-95e5-b8fe724cb4df" />

Reply:
I don't think I ever got a tree like that! Weird.